### PR TITLE
Enable metrics collection on proxy

### PR DESCRIPTION
**Pull request by paololim**
Help us better understand resources on proxy instances ahead of FN full roll out and holiday 2019

### CODE REVIEW

**Comment by benwaffle** [comment]
You'll need the aspectj java agent

**Comment by paololim** [comment]
@benwaffle ready for another look

**Comment by benwaffle** [comment]
You need `play.modules.enabled += "io.flow.play.metrics.MetricsModule"` too

File: build.sbt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import play.sbt.PlayScala._
```
Comment (benwaffle) [inline build.sbt]: i think this line isn't necessary

Comment (paololim) [reply build.sbt]: @benwaffle updated

Comment (benwaffle) [reply build.sbt]: i don't see it. missing `git push`?

Comment (paololim) [reply build.sbt]: there now

**Pull request merged**